lib/tests.js

```diff
@@ -228,6 +228,25 @@ tests.androidLongTest = async function (driver, caps) {
   }
 };
 
+tests.androidLoadTest = async function (driver, caps) {
+  let testSeconds = 600;
+  let args = await driver.elementById('textOptions');
+  let goBtn = await driver.elementById('go');
+
+  await args.clear();
+  await args.sendKeys("-M 500 -v 20" + " -s " + testSeconds);
+
+  for (let test_minutes = testSeconds / 60; test_minutes > 0; test_minutes--) {
+    await driver.sleep(30000);
+    await goBtn.click();
+
+    await driver.sleep(30000);
+    await goBtn.click();
+  }
+
+  await driver.closeApp();
```

Comment:
no need to call `closeApp`, since runsauce always calls `driver.quit` after each test anyway.

Reply:
Thanks, I'll remove that